packages/cli/src/commands/join.ts

```diff
@@ -322,8 +326,8 @@ packageVersion: string
     const componentsPrefix = getInfoPrefix(openapi.info, prefixComponentsWithInfoProp, COMPONENTS);
     if (!openapi.openapi) exitWithError('Version of specification is not found in. \n');
     if (!openapi.info) exitWithError('Info section is not found in specification. \n');
-    if (openapi.info?.description) {
-      openapi.info.description = addComponentsPrefix(openapi.info.description, componentsPrefix);
+    if (typeof openapi.info?.description === 'string') {
```

Comment:
I think we should resolve $ref and inline it into the resulting definition.

Reply:
yeah, I'll add this feature within this PR